zerver/views/user_groups.py

```diff
@@ -290,7 +311,6 @@ def notify_for_user_group_subscription_changes(
         do_send_messages(notifications)
 
 
-@transaction.atomic
```

Comment:
Not sure whether we need this now since the function calling this is already called inside a transaction to make sure the request succeeds only if both the members and subgroups are updated correctly.

Reply:
We don't; if we kept it, we'd want to switch to `savepoint=False`.